src/muya/lib/ui/formatPicker/config.js

```diff
@@ -15,12 +15,12 @@ const COMMAND_KEY = isOsx ? '⌘' : 'Ctrl'
 const icons = [
   {
     type: 'strong',
-    tooltip: 'Emphasize',
+    tooltip: 'Strong Emphasis',
     shortcut: `${COMMAND_KEY}+B`,
     icon: strongIcon
   }, {
     type: 'em',
-    tooltip: 'Italic',
+    tooltip: 'Emphasis',
```

Comment:
Personally I say keep this one as-is. I’d say “Italic” is more easily understood than “Emphasis” by most users.

Reply:
Perhaps, but the fact of emphasis being semantic features compared to bold and italic being typographic features is central to the design of MarkDown. Some users may be less intimidated as you suggest, but also realize that the icons will provide visual clues that reduce the chance of total confusion. I am not adamant on this point, but *italic* feels like the less preferable choice to me personally.

Reply:
To be fair, if there is a clearly understandable icon the change is probably okay.

Reply:
Also some editors use the term *emphasis* even without any visual indication.

Reply:
I'll keep "emphasis". If the reviewers sense a conflict with the intention, it is their prerogative to reject the change.